We do not need to check coverage for this project.

--- graphviz-0.21/pyproject.toml.orig
+++ graphviz-0.21/pyproject.toml
@@ -35,7 +35,6 @@
 test = [
   "pytest>=7,<8.1",  # https://github.com/pytest-dev/pytest/issues/12123
   "pytest-mock>=3",
-  "pytest-cov", "coverage",
 ]
 docs = ["sphinx>=5,<7", "sphinx-autodoc-typehints", "sphinx-rtd-theme>=0.2.5"]
 
@@ -71,7 +70,6 @@
   # - (p/P)assed (no output)
   "-r fsxX",
   "--durations=10",
-  "--cov", "--cov-report=term", "--cov-report=html",
   "--strict-config", "--strict-markers",
 ]
 log_cli = true
